bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/ThingManager.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2014,2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.core.thing;
+
+import org.eclipse.jdt.annotation.NonNull;
+
+/**
+ * {@link ThingManager} interface defines methods for managing a {@link Thing}.
+ * 
+ * @author Yordan Zhelev - Initial contribution
+ */
+public interface ThingManager {
+
+    /**
+     * Enable or disable the thing.
+     * 
+     * @param thingUID the {@link ThingUID} uniquely describing the {@link Thing} to be enabled or disabled.
+     * @param enabled  {@code true} if the thing should be enabled, {@code false} if the thing should be disabled.
+     */
+    void enableThing(final @NonNull ThingUID thingUID, final boolean enabled);
```

Comment:
For maximum alignment with the RuleManager, I would suggest to define exactly [these two methods](https://github.com/kaikreuzer/smarthome/blob/circular_dependencies/bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/RuleManager.java#L30-L52) here as well - feel free to copy it over (but keep using ThingUID instead of a String).